src/filesystem/boot/octopi-network.txt

```diff
@@ -9,6 +9,8 @@
 #
 # Just uncomment the lines prefixed with a single # of the configuration
 # that matches your wifi setup and fill in SSID and passphrase.
+#
+# If you need to configure more than 1 wifi networks, please use /boot/wpa_supplicant.conf instead
```

Comment:
That's still pointing to the wrong file: `wpa_supplicant.conf` instead of `octopi-wifi.txt`.

I'm still not too happy with `octopi-wifi.txt` though. Maybe `octopi-wpa-supplicant.conf` instead? 

Sorry for being so difficult about this, I just really fear people will get frustrated because they mix up the names and then create endless tickets again because they tried the syntax for `octopi-network.txt` they found somewhere in `octopi-wifi.txt` because "It says wifi!" ;)